tests/models/pix2struct/test_modeling_pix2struct.py

```diff
@@ -593,7 +593,7 @@ def test_load_vision_text_config(self):
             self.assertDictEqual(config.text_config.to_dict(), text_config.to_dict())
 
 
-# We will verify our results on an image of cute cats
+# We will verify our results on an image of a stop sign
 def prepare_img():
     url = "https://www.ilankelman.org/stopsigns/australia.jpg"
```

Comment:
Please don't use a URL from another website in the tests, better to host on HF

Reply:
Fixed, thanks for the heads up